app/styles/components/speaker-list.scss

```diff
@@ -15,12 +15,22 @@
 }
 
 .thumbnail-square {
+  height: 300px;
+  overflow: hidden;
+}
+
+.thumbnail-square img {
+  min-height: 100%;
+  object-fit: cover;
+}
+
+.featured-speaker {
   height: auto;
   overflow: hidden;
   width: 100%;
 }
 
-.thumbnail-square img {
+.featured-speaker img {
```

Comment:
Why are you changing the class, it will break every other place this class it used

Reply:
that thumbnail class is also present their. I have added new class for featured -speaker only on public page in speaker-item. 
```
.thumbnail-square {
  height: 300px;
  overflow: hidden;
}

.thumbnail-square img {
  min-height: 100%;
  object-fit: cover;
}

.featured-speaker {
  height: auto;
  overflow: hidden;
  width: 100%;
}

.featured-speaker img {
  min-height: 100%;
  min-width: 100%;
  object-fit: cover;
}
```